.gitattributes

```diff
@@ -0,0 +1 @@
+*.sh text eol=lf
```

Comment:
what does this do?

Reply:
This ensures that when checking out the repository, line endings of sh files will not change to CRLF but remain LF. This helps on Windows if the repository is checked out on Windows but the sh files being executed in WSL, since bash will otherwise fail to execute them.

Reply:
sgtm, thanks!